### PR TITLE
Add a test which checks that the transaction ID is serialised in the event

### PR DESCRIPTION
**Pull request by sandhose**
This adds a test which checks that the `transaction_id` is indeed present when querying an event with `/rooms/{roomId}/event/{eventId}`. Synapse implementation was bugged, and I fixed it in matrix-org/synapse#15174

### CODE REVIEW

File: tests/csapi/txnid_test.go [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+	"github.com/matrix-org/complement/internal/b"
+	"github.com/matrix-org/complement/internal/client"
+	"github.com/matrix-org/complement/runtime"
+	"github.com/tidwall/gjson"
+	"testing"
+)
+
+// TestTxnInEvent checks that the transaction ID is present when getting the event from the /rooms/{roomID}/event/{eventID} endpoint.
+func TestTxnInEvent(t *testing.T) {
+	// Dendrite implementation is broken
```
Comment (clokep) [inline tests/csapi/txnid_test.go:13]: If you could file an issue (and reference it here) that would be lovely!

**Review by clokep — APPROVED** [review]
Seems reasonable.

**Comment by clokep** [comment]
(Well assuming tests pass after we merge the Synapse PR. 😄 )

**Comment by clokep** [comment]
LGTM with the dendrite issue filed. 👍

File: tests/csapi/txnid_test.go (hunk 1 above)
Comment (sandhose) [reply tests/csapi/txnid_test.go:13]: Done!

**Pull request merged**